### ISSUE TITLE
button

### ISSUE THREAD

**Issue by DREAM669** [jakkra/ZSWatch#18]
I have been researching the structure of your first version recently, but I cannot understand your structure in the button section. How do you specifically implement the button function? I also noticed that you have a plastic part in the physical part of the button. Is this what you bought? Can you send me a purchase link to take a look? Thank you very much!

**Comment by jakkra** [comment]
Hi, yes buttons I experimented around with a bit and there may be multiple variants in the CAD model, the one I ended up using was this one, however it's not what I want in V2, will come up with something better. It's two parts glued together.

I'm not sure how I should solve this, would be nice with a part I could buy instead of printing it.
Or perhaps something I could print in place that would then snap off, not a bad idea...

![image](https://user-images.githubusercontent.com/4318648/232559670-5b4f1d31-a186-49fd-b708-701b5551804f.png)
